lib/MusicBrainz/Server/Entity/URL/NDLAuthorities.pm

```diff
@@ -0,0 +1,22 @@
+package MusicBrainz::Server::Entity::URL::NDLAuthorities;
+
+use Moose;
+
+extends 'MusicBrainz::Server::Entity::URL';
+with 'MusicBrainz::Server::Entity::URL::Sidebar';
+
+sub sidebar_name { 'NDL Authority' }
```

Comment:
Shouldn't it be "Web NDL Authorities"?

Reply:
Fixed in https://github.com/metabrainz/musicbrainz-server/pull/681/commits/a877864e9b1da30a40c6a1ef908c978253c50b98.